jenkins-pipelines/longevity-10gb-4h-low-load.jenkinsfile

```diff
@@ -0,0 +1,12 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    backend: 'aws',
+    region: 'eu-west-1',
+    test_name: 'longevity_test.LongevityTest.test_custom_time',
+    test_config: 'test-cases/longevity/longevity-10gb-4h-low-load.yaml',
```

Comment:
In any case, I prefer you use the 200gb-48h as the baseline (with test duration we can run it shorter than 48h).
Also here, instead of duplicating the entire yaml, you can add a configuration that overrides only the prepare_cmd to be "replication_factor=1").
(If it's mentioned in the other commands it's irrelevant anyway).

And the nemesis class to be "NonDisruptive".

Reply:
This longevity requires extensive testing with random nemesis_seed to see that it actually works as expected for all nemesis.

Reply:
Will start running it today with different nemesis seeds